dev/io.openliberty.microprofile.telemetry.1.1.internal/src/io/openliberty/microprofile/telemetry11/internal/rest/TelemetryServletFilter.java

```diff
@@ -284,9 +284,9 @@ public List<String> getHttpResponseHeader(final ServletRequest request, final Se
     private HashMap<String, String> getTelemetryProperties() {
         HashMap<String, String> telemetryProperties = new HashMap<>();
         for (String propertyName : config.getPropertyNames()) {
-
-            if (propertyName.startsWith("otel.")) {
-                config.getOptionalValue(propertyName, String.class).ifPresent(value -> telemetryProperties.put(propertyName, value));
+            if (propertyName.startsWith("otel.") || propertyName.startsWith("OTEL.") || propertyName.startsWith("otel_") || propertyName.startsWith("OTEL_")) {
+                config.getOptionalValue(propertyName.toLowerCase().replace('_', '.'), String.class).ifPresent(
+                    value -> telemetryProperties.put(propertyName.toLowerCase().replace('_', '.'), value));
             }
         }
```

Comment:
We have the same logic in the `OpenTelemetryInfoFactoryImpl` - we shouldn't need it in both places.

Raise an issue to remove this logic from here (and anywhere else where it's duplicated).

Reply:
I've raised https://github.com/OpenLiberty/open-liberty/issues/26804 for this.